packages/dashmate/configs/migrations.js

```diff
@@ -606,10 +606,12 @@ module.exports = {
       .forEach(([, config]) => {
         config.docker.network.bindIp = systemConfigs.base.docker.network.bindIp;
 
-        // eslint-disable-next-line max-len
-        config.platform.drive.tenderdash.genesis.chain_id = systemConfigs.testnet.platform.drive.tenderdash.genesis.chain_id;
-        // eslint-disable-next-line max-len
-        config.platform.drive.tenderdash.genesis.initial_core_chain_locked_height = systemConfigs.testnet.platform.drive.tenderdash.genesis.initial_core_chain_locked_height;
+        if (config.network === 'testnet') {
```

Comment:
The 15th has already been released. We need a new one.

Reply:
Yeah, there is, I just fixed the previous migration

Reply:
So you need to define a migration for 16 otherwise it won't be called.